package.json

```diff
@@ -40,18 +40,23 @@
     "simple-statistics": "^5.2.0"
   },
   "devDependencies": {
-    "@neutrinojs/airbnb-base": "^8.3.0",
-    "@neutrinojs/mocha": "^8.3.0",
-    "@neutrinojs/node": "^8.3.0",
+    "@neutrinojs/airbnb-base": "^9.0.0-rc.3",
+    "@neutrinojs/mocha": "^9.0.0-rc.3",
+    "@neutrinojs/node": "^9.0.0-rc.3",
     "coveralls": "^3.0.0",
+    "eslint": "^5",
     "fetch-mock": "^6.5.0",
     "husky": "^0.14.3",
     "lint-staged": "^7.0.5",
+    "mocha": "^6.0.0",
     "mocha-lcov-reporter": "^1.3.0",
-    "neutrino": "^8.3.0",
+    "neutrino": "^9.0.0-rc.3",
```

Comment:
For non stable versions, I would typically pin the version instead to make sure things don't break unexpectedly.